robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -39,6 +95,7 @@ def create_lookup_field(self,field_name, related):
 		self.selenium.click_element(option)
 		time.sleep(2)
 		self.selenium.click_element(next_button)
+		field_label_input = self.selenium.find_element(field_label)
```

Comment:
what is the purpose of this line? I dont see field_label_input being used anywhere

Reply:
removed